features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -127,13 +139,35 @@ internal class Base64Serializer private constructor(
             return
         }
 
-        val base64String = convertBitmapToBase64(
-            byteArray = byteArray,
-            bitmap = bitmap,
-            shouldCacheBitmap = shouldCacheBitmap
-        ).toByteArray(Charsets.UTF_8)
-        val resourceId = md5HashGenerator.generate(byteArray)?.toByteArray(Charsets.UTF_8)
-        val cacheData = CacheData(base64String, resourceId)
+        val resourceId = md5HashGenerator.generate(byteArray)
+        var base64String = "".toByteArray(Charsets.UTF_8)
```

Comment:
something is wrong here, why do we use empty string here and not a `byteArray`?

Reply:
I'm passing an empty string in the wireframe, so holding an empty string in the cache (as a bytearray). This is only necessary in this pr, as the next one will deprecate base64 and we won't need to hold a value for it at all in the cache. Even so, the name of the variable is problematic - I'll change to make this clearer